.github/workflows/build-desktop-reusable.yml

```diff
@@ -113,14 +112,17 @@ jobs:
           private_key: ${{ secrets.GH_BOT_PRIVATE_KEY }}
 
       - uses: actions/download-artifact@v4
+        if: ${{ needs.build-desktop-app.outputs.linux == 'success' }}
```

Comment:
Confirming my understanding:

- this previously causing an error at this point? So that the report creation was being halted?
- And now we do create the report but we correctly include in the report that the build for linux was a failure?